includes/discovery/functions.inc.php

```diff
@@ -382,64 +382,39 @@ function discover_sensor(&$valid, $class, $device, $oid, $index, $type, $descr,
 
 function sensor_low_limit($class, $current)
 {
-    $limit = null;
-
     switch ($class) {
         case 'temperature':
             $limit = $current - 10;
             break;
         case 'voltage':
-                $limit = $current * 0.85;
+            $limit = $current * 0.85;
             break;
         case 'humidity':
             $limit = 30;
             break;
-        case 'count':
-        case 'current':
-            $limit = null;
-            break;
         case 'fanspeed':
             $limit = $current * 0.80;
             break;
-        case 'power':
-            $limit = null;
-            break;
-        case 'power_consumed':
         case 'power_factor':
             $limit = -1;
             break;
         case 'signal':
             $limit = -80;
             break;
-        case 'airflow':
```

Comment:
airflow, dbm, snr, frequency, pressure are all supposed to be 95% of the current value.

Reply:
Updated PR to leave airflow, snr, frequency and pressure untouched. However, a 5% variance is not enough for guessed dbm limits so I kept that removed in the PR code. Detailed explanation in the updated PR description.